src/scripts/main.js

```diff
@@ -1,11 +1,31 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  return new Promise((resolve, reject) => {
+    element.addEventListener('click', (e) => {
```

Comment:
```suggestion
    element.addEventListener(`${eventName}`, (e) => {
```
Не потрібно створювати три слухача подій. Можна зробити один універсальний

Reply:
так, можна